app/src/main/java/com/github/HumanLearning2021/HumanLearningApp/presenter/DummyUIPresenter.kt

```diff
@@ -1,22 +1,29 @@
 package com.github.HumanLearning2021.HumanLearningApp.presenter
 
 import com.github.HumanLearning2021.HumanLearningApp.model.*
+import java.lang.Exception
 import java.lang.IllegalArgumentException
 
 
 /**
  * a class representing a dummy UI presenter
  */
 class DummyUIPresenter(val databaseService: DatabaseService) {
+
     /**
      * Allows to retrieve a picture fron the dummy dataset
      *
      * @param categoryName the name of the category of the picture to retrieve. Can be "knife", "fork", or "spoon"
      * @throws IllegalArgumentException if the string provided doesn't match any of "knife", "fork", or "spoon"
      */
-
     suspend fun getPicture(categoryName: String): CategorizedPicture? {
-        return databaseService.getPicture(DummyCategory(categoryName))
+        val res: CategorizedPicture?
+        try {
+             res = databaseService.getPicture(DummyCategory(categoryName, categoryName,null))
+        } catch (e: Exception) {
+            throw e
+        }
```

Comment:
Isn't this redundant?

Reply:
See https://github.com/HumanLearning2021/HumanLearningApp/pull/50#discussion_r606379453